etc/env/file_based/general.yaml

```diff
@@ -278,3 +278,6 @@ consensus:
   public_addr: "127.0.0.1:3054"
   max_payload_size: 2500000
   gossip_dynamic_inbound_limit: 100
+
+gateway_migrator:
+  eth_node_poll_interval: 300
```

Comment:
can we make the name so that duration unit is clear? Also, why does it differ from the default config?

Reply:
I want to use default config in production. And this config for testing